ksql-functional-tests/src/test/resources/query-validation-tests/asarray.json

```diff
@@ -7,7 +7,7 @@
       "name": "construct a list from two elements",
       "statements": [
         "CREATE STREAM TEST (a INT, b INT) WITH (kafka_topic='test_topic', value_format='JSON');",
-        "CREATE STREAM OUTPUT AS SELECT as_array(a, b, 3) as l FROM TEST;"
+        "CREATE STREAM OUTPUT AS SELECT ARRAY[a, b, 3] as l FROM TEST;"
```

Comment:
might want to rename this test file... it's currently `asarray.json` and `AS_ARRAY` no longer exists ;) . How about `arrays.json` :p